chevah/brink
# [#4247] PYTHON_CONFIGURATION should work for single versions.

Scope
=====

Ditto, eg. ​https://chevah.com/buildbot/builders/python-package-fbsd-10/builds/3/steps/test_1/logs/stdio, where for the compat tests, this was appended to paver.conf:
```
PYTHON_CONFIGURATION=2.7.13.f6d6244
```
As noted in https://trac.chevah.com/ticket/4244, the `default@` prefix is to be mandated. The only issue to fix is the problem with configs missing `:`, as that is not mandated.

Changes
=======

Use an array to iterate through all the versions.

**Drive-by changes**:
  * updated `paver.sh` from python-package to use 32bit binaries on SPARC.
  * updated `paver.conf` accordingly.

How to try and test the changes
===============================

reviewers: @adiroiban 

Please review the changes.
Run the automated tests and check that Solaris'es get what is needed.
Edit `PYTHON_CONFIGURATION` to only have one version, with no `:` at the end, and then check that things still work.
